src/Simulation.h

```diff
@@ -473,6 +473,9 @@ class Simulation {
 	/** use legacyCellProcessor instead of vectorizedCellProcessor */
 	bool _legacyCellProcessor = false;
 
+	/** Specifies whether to use overlapping p2p communication or not */
```

Comment:
This comment adds no info :D Extend to include the following information:
1. What overlaps?
2. What does p2p stand for?
3. Communication of what (or generally all types of communication steps)?

Reply:
```suggestion
	/**
	 * Specifies whether to use overlapping p2p (peer-to-peer) communication or not.
	 * If false: overlapping is only performed for unpacking and packing of particles.
	 * If true: overlapping of the leaving and halo particle exchange with the actual force calculation is performed.
	 */
```